exercises/bowling/bowling_test.py

```diff
@@ -0,0 +1,261 @@
+import unittest
+
+from example import BowlingGame
+
+
+class BowlingTests(unittest.TestCase):
+    def test_should_be_able_to_score_a_game_with_all_zeros(self):
+        rolls = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+
+        self.game = BowlingGame()
+
+        for roll in rolls:
```

Comment:
Probably something like:
```python
[self.game,roll(roll) for roll in rolls]
```
And the code is duplicated in every test which is not a good thing, should be refactored